ingestion/functions/parsing/brazil_rio_de_janeiro/rio_de_janeiro.py

```diff
@@ -0,0 +1,149 @@
+    'bairro' (neighborhood), which is missing in ~2/5 cases and cannot be easily geocoded with mapbox.
+    
+    
+    """
+    with open(raw_data_file, "r") as f:
+        reader = csv.DictReader(f,delimiter=';')
+        cases = []
+        for entry in reader:
+            if entry['classificacao'] and entry['dt_evento']:
+                notes = []
+                case = {
+                        "caseReference": {
+                            "sourceId": source_id,
+                            "sourceUrl": source_url
+                        },
+                        "events": [
+                        {
+                            "name": "confirmed",
+                            "dateRange":
+                            {
+                                "start": convert_date(entry["dt_evento"]),
+                                "end": convert_date(entry["dt_evento"])
+                            }
+                        }
+                    ],
+                        "location": convert_location(entry),
+                        "demographics": {
+                            "ageRange": {
+                                "start": convert_age(entry),
+                                "end": convert_age(entry)
+                            },
+                            "gender": convert_gender(entry["sexo"])
+                            }
+                    }
+                
+            if entry['dt_sintoma']:
+                case["symptoms"] = {
+                    "status": "Symptomatic",
+                }
+                case["events"].append({
+                    "name": "onsetSymptoms",
+                    "dateRange": {
+                        "start": convert_date(entry['dt_sintoma']),
+                        "end": convert_date(entry['dt_sintoma']),
+                    }
+                })
+                
+            if entry['evolucao'] == 'OBITO':
+                case["events"].append({
+                "name": "outcome",
+                "value": "Death",
+                "dateRange": {
+                    "start": convert_date(entry['dt_obito']),
+                    "end": convert_date(entry['dt_obito']),
+                    }})
+            else:
+                case["symptoms"] = {
+                    "status": "Asymptomatic",
+                }
+
+            if entry["sexo"]=='I':
+                notes.append(f'Gender not specified in case notes.')
+
+            if entry["bairro"]:
+                notes.append(f'Neighborhood of case is listed as {entry["bairro"]}')
+
+            if entry['dias']:
+                notes.append(f'Number of epidemiological days = {entry["dias"]}')
+            
+            if entry['dt_coleta_dt_notif']:
+                notes.append(f'Date case was reported was {convert_date(entry["dt_coleta_dt_notif"],dataserver=False)}')
+
+
+            case["notes"] = "\n".join(notes)
+            cases.append(case)
```

Comment:
Other parsers use ',' as the separator for notes. I don't think I need to hold up this review, but I do think we probably need to standardise.

Reply:
Yes, I agree that standardization is a good idea, and I think that ',' as the separator is safer, as `\n`, while it can make things look nicer in certain applications, in others it can cause a bit of weirdness depending on how it is interpreted.

Reply:
Agree with @calremmel on this one.

Reply:
Ok good point, will use ',' for future parsers.